FoodBookApp/Services/LocationService.swift

```diff
@@ -21,10 +21,10 @@ final class LocationService: NSObject, ObservableObject {
         self.manager.delegate = self
         self.manager.desiredAccuracy = kCLLocationAccuracyBest
         self.manager.startUpdatingLocation()
+        setupGeofence(for: uniandesLocation, regionRadius: 400, identifier: "uniandes")
```

Comment:
could you explain a bit why this change was necessary? thanks

Reply:
when testing, i realized that there was only one specific path in which the geofence was being created: the user opens the app and right away gets prompted for access and they click while using. however through testing and reinstalling the app, since those settings are saved, there wasn't a need to request access and thus the fence was not being created. this ensures that regardless of when a user accepts access to location, the geofence is created and the CAS will work as expected.

Reply:
got it, looks awesome then